routes/index.js

```diff
@@ -1,9 +1,47 @@
 const express = require('express');
+
 const router = express.Router();
 
-/* GET home page. */
-router.get('/', function(req, res, next) {
-  res.render('index', { title: 'Express' });
+const passport = require('passport');
+
+const books = require('../controller/booksController');
+
+router.get('/', (req, res, next) => {
+  res.render('login');
```

Comment:
I really recommend you to always use "return" on res.send or res.render to finish the code execution....

`return   res.render('login');`